Dockerfile

```diff
@@ -29,17 +100,13 @@ ENV \
   # https://nodejs.org/api/cli.html#cli_title_title
   NODE_OPTS='--title=digitransit-ui' \
   RELAY_FETCH_TIMEOUT='' \
+  WEBPACK_DEVTOOL=$WEBPACK_DEVTOOL \
   ASSET_URL='' \
   STATIC_MESSAGE_URL=''
 
-WORKDIR ${WORK}
-ADD . ${WORK}
-
-RUN \
-  yarn && \
-  yarn setup && \
-  yarn build && \
-  rm -rf static docs test /tmp/* .cache && \
-  yarn cache clean --all
+RUN apk add --no-cache curl
+HEALTHCHECK \
```

Comment:
I wonder what this healthcheck does in practice. If we have configured different health checks in kubernetes, what happens?

Reply:
The health check is picked up by e.g. the regular Docker CLI (`docker run`), Docker Swarm, and various other container-based tools.

[Kubernetes ignores it if there is a readiness/liveness check defined](https://stackoverflow.com/a/41476481).